README.md

```diff
@@ -107,6 +107,7 @@ All modules are disabled by default and need to be activated explicitly in your
 lua <<EOF
 require'nvim-treesitter.configs'.setup {
   ensure_installed = "maintained", -- one of "all", "maintained" (parsers with maintainers), or a list of languages
+  sync_install = false, -- install languages synchronously
```

Comment:
this only refers to ensure_installed. but the name suggest and the comment suggest that it would affect installation in general (which it does not). Wouldn't it be better to have `ensure_installed` and `ensure_installed_static` both as properties?

You could also extend the comment that this only refers to `ensure_installed`

Reply:
At first I thought to introduced 2 separated properties as you suggests. But that would also means we might have to explain on what would be the outcome if the user has both properties in their configurations, hence introducing a boolean here.

New name would be much appreciated here, as I cannot find a better name that concise and to the point to this (beside just adding a comment). Meanwhile, I'll update the comment here to clearly explain that it applied to `ensure_installed` only.

Reply:
Yes, that sounds reasonable (extending the comment). Well, when both properties would be specified I would expect that it would do the corresponding action for the lists specified in the two commands and just any of the two when languages are in both of the properties. But let's stay with extending the comment.